store/store.go

```diff
@@ -87,7 +91,28 @@ func (s *Store) Start() error {
 	if err != nil {
 		return err
 	}
-	return s.TxManager.ActivateAccount(acc)
+
+	err = s.TxManager.ActivateAccount(acc)
+
+	var transactions []models.Tx
+	query := s.Select(q.Eq("From", acc.Address))
+	if err = query.Limit(1).OrderBy("Nonce").Reverse().Find(&transactions); err == storm.ErrNotFound {
+		return nil
+	}
+
+	if localNonceIsNotCurrent(transactions, s.TxManager.activeAccount.GetNonce()) {
```

Comment:
I'm pretty conflicted about pinging the blockchain on `Store#Start`. It's a pretty fundamental piece of our system that gets invoked multiple times. Perhaps there's another spot we can display this warning, such as `cmd/client.go`.

Example:

https://github.com/smartcontractkit/chainlink/blob/master/cmd/client.go#L57

Reply:
yes i agree